frontend/src/components/elements/ImageList/ImageList.tsx

```diff
@@ -36,7 +36,7 @@ enum WidthBehavior {
   ColumnWidth = -2,
 }
 
-function getImageURI(imgProto: ImmutableMap<string, any>): string {
+export function getImageURI(imgProto: ImmutableMap<string, any>): string {
```

Comment:
This looks like a `utils.ts` function. But feel free to leave it here if you believe this is a better place.

Reply:
Extracted as a  UriUtil function (and consolidated logic in Audio.tsx, Video.tsx too)